assets/js/frontend/give-ajax.js

```diff
@@ -132,7 +132,7 @@ jQuery(document).ready(function ($) {
 	 *
 	 * @description: Process the donation submit
 	 */
-	$('body').on('click touchend', 'form.give-form input[type="submit"].give-submit', function (e) {
+	$('body').on('click touchend', 'form.give-form input[name="give-purchase"].give-submit', function (e) {
```

Comment:
@DevinWalker Check this. This event was also firing for login button click, so I updated it to more specific ( for donate button)